static/docs/commands-reference/import.md

```diff
@@ -2,15 +2,6 @@
 
 Import file from URL to local directory and track changes in remote file.
```

Comment:
local workspace or working space? URL can pattern match with HTTP only. Can we specify somehow that it can be any remote resource - s3, gs, etc?